vulkanDeviceInfo.cpp

```diff
@@ -596,7 +596,10 @@ QJsonObject VulkanDeviceInfo::toJson(std::string fileName, std::string submitter
     jsonSurfaceCaps["validSurface"] = bool(surfaceInfo.validSurface);
     if (surfaceInfo.validSurface)
     {
-        jsonSurfaceCaps["surfaceExtension"] = QString::fromStdString(surfaceInfo.surfaceExtension);
+        QJsonArray surfaceExtensions;
+        for (std::string surfaceExtension : surfaceInfo.surfaceExtensions)
+          surfaceExtensions.append(QString::fromStdString(surfaceExtension));
+        jsonSurfaceCaps["surfaceExtensions"] = surfaceExtensions;
```

Comment:
This won't work with the current database layout. The api route to store reports in the database only acccepts one surface extension and it has to be stored in a json attribute named "surfaceExtension". To make this work I'd suggest to revert to only storing a single surface extension for now.

Reply:
Ah, I forgot about that.

I'll rewrite so that we only store the list locally and once the surface is created we store that surface's extension to be sent to the server, does that work?